examples/grasping_client.py

```diff
@@ -48,10 +47,10 @@
 from gqcnn.msg import GQCNNGrasp, BoundingBox
 from gqcnn.srv import GQCNNGraspPlanner
 
-if __name__ == '__main__':
-    # set up logger
-    logging.getLogger().setLevel(logging.INFO)
+# set up logger
+logger = Logger.get_logger('examples/grasping_client.py')
```

Comment:
Is this sensitive to relative paths? If so we need to fix. You can use os.path.dirname(os.path.realpath(__file__))

Reply:
No, this is just a name for the logger-it's not a path.

Reply:
Yes, the loggers for tools & examples are named {tools,examples}/{name of file}. I thought that was descriptive enough to find the file if needed for debugging.